crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -319,7 +332,9 @@ impl CompressedDataManager {
         let mut compressed_data_buffer = self
```

Comment:
Maybe replace with `let (_key, mut compressed_data_buffer) = self` to make the code a bit clearer.

Reply:
Fixed.